lib/csv.rb

```diff
@@ -882,6 +886,33 @@ def filter(input=nil, output=nil, **options)
     #   warning: Unsupported encoding foo ignored
     #   warning: Unsupported encoding bar ignored
     #
+    # ====== With Option +headers+
+    #
+    # With option +headers+, returns each row as a CSV::Row object.
```

Comment:
May be worthy to mention that `headers` requires a truly value to yield `CSV::Row` objects, in the other hand, a falsie  value will behave as no `headers` parameter: `CSV.foreach(path, headers: nil)`

Reply:
Thanks, @esparta.  I'll review this.  I didn't want to say "set," b/c that's vague.  Maybe "truthy value" and link to the section "Option headers."